src/Microsoft.Dnx.TestHost/Project.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright(c) .NET Foundation.All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Newtonsoft.Json;
+
+namespace Microsoft.Dnx.TestHost
+{
+    public class Project
+    {
+        private const string ProjectFileName = "project.json";
+
+        public string Name { get; set; }
+
+        public IDictionary<string, string> Commands { get; }
+            = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
+
+        public string EntryPoint { get; set; }
```

Comment:
What is an entry point? I can guess what it is but I couldn't find any information for it in http://json.schemastore.org/project or from [ProjectReader] (https://github.com/aspnet/dnx/blob/dev/src/Microsoft.Dnx.Runtime/ProjectReader.cs#L78)...there seem to be no tests for it...is it dead code?
@troydai